functions/Import-DbaCertificate.ps1

```diff
@@ -41,62 +43,78 @@ Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
 License: GNU GPL v3 https://opensource.org/licenses/GPL-3.0
 
 .EXAMPLE
-Import-DbaCertificate -SqlServer Server1 -Path \\Server1\Certificates -password (ConvertTo-SecureString -force -AsPlainText GoodPass1234!!)
+Import-DbaCertificate -SqlInstance Server1 -Path \\Server1\Certificates -password (ConvertTo-SecureString -force -AsPlainText GoodPass1234!!)
 Imports all the certificates in the specified path.
 
-.EXAMPLE
-Import-DbaCertificate -SqlServer Server1 -Path \\Server1\Certificates -Certificates "CertTDE"
-Prompts for password then imports certificate in the specified path named 'CertTDE'
-
 #>
 	[CmdletBinding(DefaultParameterSetName = "Default", SupportsShouldProcess = $true)]
 	param (
 		[Parameter(Mandatory = $true)]
 		[ValidateNotNullOrEmpty()]
-		[Alias("ServerInstance","SqlInstance")]
-        [object]$SqlServer,
-        [parameter(Mandatory = $true, ValueFromPipeline = $true)]
+		[Alias("ServerInstance", "SqlServer")]
+		[object]$SqlInstance,
+		[System.Management.Automation.PSCredential]$SqlCredential,
+		[parameter(Mandatory = $true, ValueFromPipeline = $true)]
 		[object[]]$Path,
-        [System.Management.Automation.PSCredential]$SqlCredential,
-		[Array]$Certificates,
-		[Security.SecureString] $Password = (Read-Host "Password" -AsSecureString),
-		[switch]$Silent	
+		[string]$Database = "master",
+		[Security.SecureString]$Password = (Read-Host "Password" -AsSecureString),
+		[switch]$Silent
 	)
-
-	BEGIN {
-		$server = Connect-SqlServer $SqlServer $SqlCredential
+	
+	begin {
+		
+		function new-smocert ($directory, $certname) {
+			if ($Pscmdlet.ShouldProcess("$cert on $SqlInstance", "Importing Certificate")) {
+				$smocert = New-Object Microsoft.SqlServer.Management.Smo.Certificate
+				$smocert.Name = $certname
+				$smocert.Parent = $server.Databases[$Database]
+				Write-Message -Level Verbose -Message "Creating Certificate: $certname"
+				try {
+					$fullcertname = "$directory\$certname.cer"
+					$privatekey = "$directory\$certname.pvk"
+					Write-Message -Level Verbose -Message "Full certificate path: $fullcertname"
+					Write-Message -Level Verbose -Message "Private key: $privatekey"
+					$smocert.Create($fullcertname, 1, $privatekey, [System.Runtime.InteropServices.marshal]::PtrToStringAuto([System.Runtime.InteropServices.marshal]::SecureStringToBSTR($password)))
+				}
+				catch {
+					Write-Message -Level Warning -Message $_ -ErrorRecord $_ -Target $instance
+				}
+			}
+		}
+		
+		try {
+			Write-Message -Level Verbose -Message "Connecting to $SqlInstance"
+			$server = Connect-SqlServer -SqlServer $SqlInstance -SqlCredential $sqlcredential
+		}
+		catch {
+			Stop-Function -Message "Failed to connect to: $SqlInstance" -Continue -Target $SqlInstance -InnerErrorRecord $_
```

Comment:
Wrong usage. Don't use the `-Continue` parameter outside a loop.
Use `return` on next line instead